tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -122,19 +122,32 @@ async def test_reborn_v2_text_turn_persists(reborn_v2_server):
         )
 
 
-async def test_reborn_v2_ui_send_renders_reply(reborn_v2_page, reborn_v2_server):
-    """Typing in the composer and pressing Enter renders the assistant reply in the SPA."""
+async def test_reborn_v2_ui_enter_submits_initial_and_follow_up_messages(
+    reborn_v2_page, reborn_v2_server
```

Ruff (0.15.21)

[warning] Unused function argument: `reborn_v2_server`

(ARG001)